part-iii-infrastructure-components/event-processing.md

```diff
@@ -100,6 +106,19 @@ SagaConfiguration.trackingSagaManager(MySaga.class)
 
 The `TrackingProcessingConfiguration` has a few methods allowing you to specify how many segments will be created and which ThreadFactory should be used to create Processor threads. See [Parallel Processing](event-processing.md#parallel-processing) for more details.
 
+If you are using Spring, you can configure your saga like this:
+
+```java
+@Saga(configurationBean = "mySagaConfiguration")
+public class Saga {...}
+...
+// somewhere in configuration
+@Bean
+public SagaConfiguration mySagaConfiguration() {
+	// return saga configuration
```

Comment:
instead of the comment, just create a bare example of a SagaConfiguration instance.